### DISCUSSION TITLE
Will there be a compatible version port to 1.20.4

### DISCUSSION THREAD

**Discussion by Piester37** [cc-tweaked/CC-Tweaked#1694] [Q&A] [answered]
I usually wonder if you are going to port this to 1.20.4. Soooooooooooo
Can you port it to port to version 1.20.4, if can't, tell me reason.
-piester

**Comment by SquidDev** [comment, marked as answer]
This is currently being worked on in #1618. It's just a slow process, as both Minecraft and the underlying mod loaders have changed quite a lot.
